typescript/sdk/src/ism/metadata/aggregation.ts

```diff
@@ -0,0 +1,62 @@
+import { fromHexString, toHexString } from '@hyperlane-xyz/utils';
+
+// null indicates that metadata is NOT INCLUDED for this submodule
+// empty or 0x string indicates that metadata is INCLUDED but NULL
+export interface AggregationIsmMetadata {
+  submoduleMetadata: Array<string | null>;
+}
+
+const RANGE_SIZE = 4;
+
+// adapted from rust/agents/relayer/src/msg/metadata/aggregation.rs
+export class AggregationIsmMetadataBuilder {
```

Comment:
I'm not suggesting you change this because I realize the SDK is already super class-heavy but if a class 
1. has no state
2. has all static methods
3. isn't using inheritance in some way

Then it's not a class, it's a namespace and could have been an object. Just food for thought.

Reply:
agreed
this PR was bisected from code where this was not just static